pgml-extension/sql/pgml--2.8.1--2.8.2.sql

```diff
@@ -25,3 +25,109 @@ CREATE  FUNCTION pgml."deploy"(
 AS 'MODULE_PATHNAME', 'deploy_strategy_wrapper';
 
 ALTER TYPE pgml.strategy ADD VALUE 'specific';
+
+-- src/orm/sampling.rs:6
+-- pgml::orm::sampling::Sampling
+DROP TYPE IF EXISTS pgml.Sampling;
+CREATE TYPE pgml.Sampling AS ENUM (
+	'random',
+	'last',
+	'stratified'
+);
```

Comment:
I still need to do some more testing to make sure this migration works as intended.

@montanalow , are there integration tests that assert migrations work? I didn't see any...

Reply:
There aren't integration tests. I typically run the test/test.sql to populate a database with the previous version, then `later extension pgml update`.

Reply:
You'll want to `alter type add value...` here rather than dropping the enum, as that would only work on an empty database.